nautobot/core/api/serializers/__init__.py

```diff
@@ -139,37 +139,6 @@ def get_field_names(self, declared_fields, info):
         return fields
 
 
-class ValidatedModelSerializer(BaseModelSerializer):
```

Comment:
Disregard changes to this file (other than converting it into a submodule) - the relocation of `ValidatedModelSerializer` within this file are leftovers from some early refactoring that proved unnecessary.